src/main/java/pl/wojtczak/lesson5/homework/Employee.java

```diff
@@ -0,0 +1,66 @@
+package pl.wojtczak.lesson5.homework;
+
+import java.math.BigDecimal;
+
+public class Employee {
+
+    private final String name;
+    private final String surname;
+    private int age;
+    private BigDecimal baseSalary;
+    private BigDecimal bonus;
+
+
+    public Employee(String name, String surname) {
+        this.name = name;
+        this.surname = surname;
+    }
+
+    public Employee(String name, String surname, int age) {
+        this.name = name;
+        this.surname = surname;
+        this.age = age;
+
+    }
+
+    public Employee(String name, String surname, int age, BigDecimal baseSalary) {
+        this.name = name;
+        this.surname = surname;
+        this.age = age;
+        this.baseSalary = baseSalary;
+
+    }
+
+    public Employee(String name, String surname, int age, BigDecimal baseSalary, BigDecimal bonus) {
+        this.name = name;
+        this.surname = surname;
+        this.age = age;
+        this.baseSalary = baseSalary;
+        this.bonus = bonus;
+
+    }
+
+
+    public String getEmployeeInfo() {
+        return "Employee name : " + name + " " + surname;
+    }
+
+    public String getEmployeeDetails() {
+        return "Employee details : " + name + " " + surname + " is " + age;
+    }
+
+
+    // Method is used to get total  salary
```

Comment:
a ten komentarz to po co nam?

Reply:
czy on masz tutaj jakąś wartość? przecież nazwa metody 'getSalaryAndBonus' mówi tyle samo